Author: Julian Taylor <jtaylor.debian@googlemail.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/ipython/+bug/241535
Subject: use LightBG colorscheme by default
 Linux scheme leads to invisible text when console background is light

Index: ipython/IPython/UserConfig/ipy_user_conf.py
===================================================================
--- ipython.orig/IPython/UserConfig/ipy_user_conf.py	2011-04-21 17:54:49.858160694 +0200
+++ ipython/IPython/UserConfig/ipy_user_conf.py	2011-04-21 17:54:58.658160688 +0200
@@ -68,7 +68,7 @@
     
     # Try one of these color settings if you can't read the text easily
     # autoexec is a list of IPython commands to execute on startup
-    #o.autoexec.append('%colors LightBG')
+    o.autoexec.append('%colors LightBG')
     #o.autoexec.append('%colors NoColor')
     #o.autoexec.append('%colors Linux')
     
Index: ipython/IPython/UserConfig/ipythonrc
===================================================================
--- ipython.orig/IPython/UserConfig/ipythonrc	2011-04-21 17:55:33.928160670 +0200
+++ ipython/IPython/UserConfig/ipythonrc	2011-04-21 17:55:40.068160666 +0200
@@ -163,8 +163,8 @@
 # in light background terminals.
 
 # keep uncommented only the one you want:
-colors Linux
-#colors LightBG
+#colors Linux
+colors LightBG
 #colors NoColor
 
 ########################
Index: ipython/IPython/Extensions/ipy_defaults.py
===================================================================
--- ipython.orig/IPython/Extensions/ipy_defaults.py	2011-04-21 18:07:16.368160277 +0200
+++ ipython/IPython/Extensions/ipy_defaults.py	2011-04-21 18:07:21.588160275 +0200
@@ -18,7 +18,7 @@
 
 o = ip.options
 
-o.colors = "Linux"
+o.colors = "LightBG"
 o.color_info=1
 o.confirm_exit=1
 o.pprint=1
